src/app/Viewport/Viewport.jsx

```diff
@@ -6,19 +6,35 @@
 
 import React, { useEffect, useRef } from "react";
 import * as THREE from "three";
-import FPSMonitor from "../commons";
+import { makeStyles } from "@material-ui/core/styles";
+import ContorlBar from "./components/ContorlBar";
 import { nanocar } from "../../data";
 import { addMolecule } from "../../utils";
 
 // get OrbitControls
 const OrbitControls = require("three-orbitcontrols");
 
+/**
```

Comment:
I'm not a fan of this approach. So, it's easy to lose the context of complex layouts. What do you think for separate js file for styles?
```
Viewport.jsx
Viewport.style.jsx
```

Reply:
My decision to put the style inside of the component was because it doesn't contain many styles. If  styles would start to grow, I would separate it from the component.

I believe you are right. Also it will be nice as a structural example for future contributors.

Will make the necessary changes.

Reply:
My decision to put the style inside of the component was because it doesn't contain many styles. If  styles would start to grow, I would separate it from the component as I did in ControlBar component.

I believe you are right. Also it will be nice as a structural example for future contributors.

I will make the necessary changes.